### PR TITLE
extendable config for dcos-history

### PR DESCRIPTION
**Pull request by alberts**

### CODE REVIEW

File: packages/dcos-history/extra/dcos-history.service [hunk 1]
```diff
@@ -8,7 +8,8 @@ Restart=always
 StartLimitInterval=0
 RestartSec=5
 EnvironmentFile=/opt/mesosphere/environment
-EnvironmentFile=-/run/dcos/etc/history-service/history-service.env
```
Comment (cmaloney) [inline packages/dcos-history/extra/dcos-history.service]: Pretty sure the extension still needs runtime extension and not just config extension.

Comment (alberts) [inline packages/dcos-history/extra/dcos-history.service]: No need for runtime extension. See https://github.com/mesosphere/dcos-enterprise/pull/185/files#diff-8acc8e77a15678f67a353d405fcf971cR549

**Comment by alberts** [comment]
@cmaloney please add this one to your next train. test failures are all unrelated flakiness

**Comment by cmaloney** [comment]
:ship:

**Pull request merged**